backend/object/posix.c

```diff
@@ -315,7 +315,7 @@ backend_read (gpointer data, gpointer buffer, guint64 length, guint64 offset, gu
 		{
 			break;
 		}
-		else if (nbytes < 0)
+		if (nbytes < 0)
```

Comment:
Why?

Reply:
The else was removed by an automated tool, because the break in the "true"-branch makes the "else" useless.